workflows/mag.nf

```diff
@@ -463,20 +454,63 @@ workflow MAG {
                                     Taxonomic information
     ================================================================================
     */
-    CENTRIFUGE_DB_PREPARATION ( ch_centrifuge_db_file )
+    if ( !ch_centrifuge_db_file.isEmpty() ) {
+        if ( ch_centrifuge_db_file.extension == "gz" || ch_centrifuge_db_file.extension == "tgz" ) {
```

Comment:
if(ch_centrifuge_db_file.extension in [“gz”, “tgz”]{ ? A bit less verbose

Reply:
Done